src/components/Movie.jsx

```diff
@@ -0,0 +1,72 @@
+import { useDispatch, useSelector } from 'react-redux'
+import starredSlice from '../data/starredSlice'
+import watchLaterSlice from '../data/watchLaterSlice'
+import placeholder from '../assets/not-found-500X750.jpeg'
+
+const Movie = ({ movie, viewTrailer, closeCard }) => {
```

Comment:
- To keep things simple and organized, we should split up the logic for marking as favorite and watch later into their own components.
- Create a StarButton.jsx component.
- Create a WatchLaterButton.jsx component.
- Let the Movie component act as a container to organize these buttons.